include/revng/Support/revng.h

```diff
@@ -14,7 +14,7 @@
 #include "llvm/ADT/Triple.h"
 #include "llvm/BinaryFormat/ELF.h"
 
-#include "revng/Model/ABI.h"
+#include "revng/Model/Generated/Early/ABI.h"
```

Comment:
Mmmmmmmmh

Reply:
Ops, leftover from early times when enums did not have a manually written associated class, replaced with `#include "revng/Model/ABI.h"`